prow/gerrit/client/client.go

```diff
@@ -179,14 +179,26 @@ func (l LastSyncState) DeepCopy() LastSyncState {
 	return result
 }
 
+type roundTripperWithHeader struct {
+	upstream http.RoundTripper
```

Comment:
Nit: Why not just `roundTripper` for the field name instead of `upstream`?

Reply:
> Nit: Why not just `roundTripper` for the field name instead of `upstream`?

copied from other places in this codebase :)